spec/intergration/post_show_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+require 'capybara/rspec'
+RSpec.describe Post, type: :system do
+  user = User.create(name: 'Tom', posts_counter: 3, photo: 'https://pic.com',
```

Comment:
Good work implementing some tests! 

- [ ] Kindly fix the error in the terminal


![image](https://github.com/Roland-Ntwali/Blog-App/assets/93630700/53e62fb1-1044-440e-940a-bbb14b9107c2)